### PR TITLE
Improve on OTP 21 compatibility

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**

### CODE REVIEW

**Comment by paulo-ferraz-oliveira** [comment]
I ended up making changes regarding xref and OTP 19+ but didn't rename the branch afterward.

**Comment by g-andrade** [comment]
👍

File: src/uuid.erl [hunk 1]
```diff
@@ -30,6 +30,7 @@
 %
 -module(uuid).
 -export([v4/0, to_string/1, get_parts/1, to_binary/1]).
+-ignore_xref([{v4, 0}, {to_string, 1}, {get_parts, 1}, {to_binary, 1}]).
```
Comment (g-andrade) [inline src/uuid.erl:33]: Same syntax as in "-export" can be used, btw (e.g. "-ignore_xref([v4/0, to_string/1, ...])."

Comment (paulo-ferraz-oliveira) [reply src/uuid.erl:33]: 👍

**Comment by g-andrade** [comment]
Oh man, at the time I missed the bad usage of `rand:uniform/1`. It has been fixed on PR #3 .

**Pull request merged**